CHANGELOG.md

```diff
@@ -8,3443 +10,1341 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
ncDecRector, Thanks to [@samsonasik]- [#5072] [CodingStyle] Add %s for pass PHP_EOL constant on EncapsedStringsToSprintfRector, Thanks to [@samsonasik]- [#5015] [CodingStyle] Add UnSpreadOperatorRector, Thanks to [@samsonasik]- [#4931] [DX] Add Interactive Mode to Generate command, Thanks to [@simivar]- [#4883] [DX] Add ValidateFixtureNamespaceCommand and ValidateFixtureClassnameCommand, Thanks to [@samsonasik]- [#4889] [Downgrade] Add ".0" after downgrading the numeric literal separator on floats, Thanks to [@leoloso]- [#4974] [EarlyReturn] Add ChangeOrIfReturnToEarlyReturnRector, Thanks to [@samsonasik]- [#4906] [EarlyReturn]  Add ReturnBinaryAndToEarlyReturnRector, Thanks to [@samsonasik]- [#4863] [Naming] add failing test case for type property collection name- [#5065] [NetteToSymfony] Add new Route class support- [#4998] [PHPStan] Added PathsAreNotTooLongRule phpstan rule- [#5043] [PHPStanStaticTypeMapper] Add closure conversion- [#5031] [PhpVersionResolver] Add project composer.json as source + decouple to composer json fatcory- [#5034] [Test] Add utils/ to whitelist test coverage, Thanks to [@samsonasik]- [#4907] [UPGRADE] Add Rector 0.9 upgrade docs- [#4846] [Symfony 5.2] Add BinaryFileResponseCreateToNewInstanceRector, Thanks to [@simivar]- [#4874] [Symfony 5.2] Add PropertyAccessorCreationBooleanToFlagsRector, Thanks to [@simivar]- [#4806] [Downgrade][PHP 7.3] Add SetCookieOptionsArrayToArgumentsRector, Thanks to [@simivar]- [#4807] [Downgrade PHP 8.0] Fixes [#4154] Add DowngradeNullsafeToTernaryOperatorRector, Thanks to [@samsonasik]- [#4973] add phpstan-for-rector config path, consolidate return type extensions - re-use from Symplify- [#5068] [FlySystem 2.0] Add Upgrade set for MethodCallRename, Thanks to [@samsonasik]- [#4815] Added test cases for RenameParamToMatchTypeRector, Thanks to [@leoloso]- [#5013] Add failing test fixture for AddArrayReturnDocTypeRector - array shape is replaced by structural type, Thanks to [@jkuchar]- [#4821] [Downgrade PHP 7.1] Add SymmetricArrayDestructuringToListRector, Thanks to [@samsonasik]- [#4823] [Downgade PHP 7.1] Add DowngradeNegativeStringOffsetToStrlenRector, Thanks to [@samsonasik]- [#5019] Add template annotation resolving support- [#5020] Add support for inheritdoc in parent type use- [#4898] Added tests for RenameVariableToMatchNewTypeRector, Thanks to [@leoloso]- [#4825] [Downgrade PHP 7.4]  Add DowngradeFreadFwriteFalsyToNegationRector, Thanks to [@samsonasik]- [#4981] Add Visibility consts- [#4828] [Downgrade PHP 8.0]  Add DowngradeTrailingCommasInParamUseRector, Thanks to [@samsonasik]- [#4876] [Symfony 5.2] Add ReflectionExtractorEnableMagicCallExtractorRector, Thanks to [@simivar]- [#4887] [Symfony 5.2] Add PropertyPathMapperToDataMapperRector, Thanks to [@simivar]- [#4979] [Util] Add StaticInstanceOf, Thanks to [@samsonasik]- [#4814] [Utils] Add OnlyOneClassMethodRule- [#4996] [cs] add variable types- [#4997] [scoper] adding Symfony version test, name prefix by date for easier diffs
```

Comment:
@TomasVotruba the re-generate changelog somehow trim the new line. It seems happen on symplify changelog too https://github.com/symplify/symplify/blob/master/CHANGELOG.md#phpstanrules

Reply:
It might be related to the unreleased change. 
Could you verify?

Reply:
I will try